source/_integrations/reolink.markdown

```diff
@@ -204,3 +204,4 @@ However, these cameras can work with this integration through an NVR in which th
 - Older firmware versions do not expose the necessary information the integration needs to function. Ensure the camera is updated to the [latest firmware](https://reolink.com/download-center/) prior to setting up the integration. Note that Reolink auto update and check for update functions in the app/windows/web client often do not show the latest available firmware version. Therefore check the version in the [Reolink download center](https://reolink.com/download-center/) online.
 - Ensure at least one of the HTTP/HTTPS ports is enabled in the [windows](https://reolink.com/software-and-manual/)/web client under `Settings`->`Network`->`Advanced`->`Port Settings`, see [additional instructions](https://support.reolink.com/hc/en-us/articles/900004435763-How-to-Set-up-Reolink-Ports-Settings-via-Reolink-Client-New-Client-) on the Reolink site.
 - On some camera models, the RTMP port needs to be enabled in order for the HTTP(S) port to function properly. Make sure this port is also enabled if you get a `Cannot connect to host` error while one of the HTTP/HTTPS ports is already enabled.
+- There is a known issue with the firmware of Reolink cameras not supporting webhooks behind a HTTPS endpoint. Home Assistant instances that are only available to your camera over HTTPS (i.e. both the internal and external interface of Home Assistant are configured with certificates and use HTTPS) then certain webhooks will not work. For example, the Visitor binary sensor from the Video Doorbell will not be updated when the doorbell button is pressed.
```

Comment:
In the suggestion:

- It is not an issue but a limitation. Documenting it as an issue may seem like it is unresolvable or should be reported.

```suggestion
- Reolink cameras not supporting webhooks behind an HTTPS endpoint. Home Assistant instances that are only available to your camera over HTTPS (for example, both the internal and external interface of Home Assistant are configured with certificates and use HTTPS), then certain webhooks will not work. For example, the visitor binary sensor from the Video Doorbell will not be updated when the doorbell button is pressed.
```

I do still have a concern with this line, as it is unclear/ambiguous. I think you are trying to explain this is an issue when the Home Assistant instance is configured to use SSL in the `http` integration?

Also, what is the action a user has to take?

Reply:
> There are work arounds, but they are in the realm of advanced/expert users and should certainly not be documented here :-)

Is it? The upcoming repairs issue will instruct users to use a reverse proxy by using an add-on.

Reply:
Sure, I got that, my last comment did not say anything like that?

Reply:
How do you wish to progress? I've put forward a PR to help others out with an issue which is current. As best I can tell there is nothing that can be done in the current release to resolve this without telling users to edit code and set up a reverse proxy.

Short of getting Reolink to do something at the firmware side of things I don't know how to progress this now. Please provide a suggestion.

Reply:
As frenck mentioned, HomeAssistant will offer repair issues with instructions on how to configure HomeAssistant diffrently (Use HTTP internally and use a reverse proxy addon for HTTPS externally) in order to get Reolink working properly in case a situation is detected in which the Reolink integration can not properly function.

Therefore personnally I don't think a extra warning needs to be present in the documentation, since that might confuse people that do not have any issues at all.

If we add this, we schould not say "certain webhooks will not work" but explicitly state what will and what will not work: everything will work except for the binary sensors: motion, people, face, vehicle, pet and visitor (detection).

Reply:
@craiggenner maybe you could review PR https://github.com/home-assistant/core/pull/91597 since that will be the last missing peace to inform users about network configuration issues.